test/conftest.py

```diff
@@ -61,6 +62,45 @@ def pytest_addoption(parser):
     )
 
 
+@pytest.hookimpl(tryfirst=True)
+def pytest_runtest_makereport(item, call):
+    """Make timeouts appear as such in the logs.
+
+    The pytest-timeout plug-in messes with the durations that are reported for tests.
+    Fix that by marking the report such that we can print `TIMEOUT` instead of
+    `FAILED (incorrect duration)`. See
+    https://github.com/pytest-dev/pytest/blob/38d8deb74d95077ebf189440ca047e14f8197da1/src/_pytest/runner.py#L366
+    """
```

Comment:
If this is the case, how do others who use pytest-timeout handle this?

Reply:
Haven't had a deep look round, but most people just use it to fail tests after a certain period. Most users don't care about timings AND time-outs. Printing TIMEOUT was the only solution that didn't involve printing incorrect durations. Even with `time.time` in the exception handler I got weird results that were below the threshold.

Reply:
OK 👍🏽